src/biome/text/pipeline.py

```diff
@@ -177,6 +179,8 @@ def train(
         epoch_callbacks:
             A list of callbacks that will be called at the end of every epoch, and at the start of
             training (with epoch = -1).
+        loggers:
+            A list of training loggers used in training. See `MlflowLogger` as example
```

Comment:
Maybe we could explain this component a bit more in detail, something like:
```suggestion
            A list of loggers that execute a callback before the training, after each epoch, and at the end of the training (see `biome.text.logger.MlflowLogger`, for example)```